### PR TITLE
Fractionation - FHIR-37976

### PR DESCRIPTION
**Pull request by saulakravitz**
Added a boolean extension uniformFractionation to totalDoseDelivered extension in Course Summary

### CODE REVIEW

File: input/fsh/SD_Extensions.fsh [hunk 1]
```diff
@@ -181,6 +181,15 @@ Description: "Radiotherapy Treatment Applicator Type."
 * value[x] only CodeableConcept
 * value[x] from ApplicatorTypesVS (extensible)
 
+Extension: UniformFractionation
+Id: codexrt-radiotheraphy-uniform-fractionation
+Title: "Uniform Fractionation"
+Description: "Uniform Fraction is true if fractionation was unform, false if fractionation was mixed."
```
Comment (mvonsieb) [inline input/fsh/SD_Extensions.fsh]: typo: "unform" Maybe add more text that "uniform" here means with same modality, and dose per fraction?

File: input/fsh/EX_TreatmentLocation.fsh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+InstanceOf: USCoreLocation
+Title: "Treatment Location 1"
+Description: "An example treatment location."
+Usage: #example
+* id = "Treatment-Location-1"
+* meta.security = http://terminology.hl7.org/CodeSystem/v3-ActReason#HTEST
+* type = v3-RoleCode#RADDX "Radiology diagnostics or therapeutics unit"
+* name = "Joe's Radiotherapy Clinic"
+* address
+  * state = "MN"
```
Comment (mvonsieb) [inline input/fsh/EX_TreatmentLocation.fsh]: Should we invent a full address here as the requirement was to cover Institution name and address?

Comment (saulakravitz) [reply input/fsh/EX_TreatmentLocation.fsh]: added a city/state/street address

File: input/fsh/SD_Extensions.fsh (hunk 1 above)
Comment (saulakravitz) [reply input/fsh/SD_Extensions.fsh]: looks like you added it...

**Pull request merged**